docs/src/examples/redis-clustering-aware.md

```diff
@@ -0,0 +1,156 @@
+# Redis Clustering with cluster aware client
+
+The following guide shows you how to configure Shotover to support proxying Redis cluster *aware* clients to [Redis cluster](https://redis.io/topics/cluster-spec).
+
+## Overview
+
+In this example, we will be connecting to a Redis cluster that has the following topology:
+
+* `127.0.0.1:7000`
+* `127.0.0.1:7001`
+* `127.0.0.1:7002`
+* `127.0.0.1:7003`
+* `127.0.0.1:7004`
+* `127.0.0.1:7005`
+
+Shotover will listen on the loopback adapter and act as a sidecar for our application that speaks Redis. In this example we will use `redis-benchmark` with cluster mode enabled as our Redis cluster aware client application. Use the following [docker-compose.yml](https://github.com/shotover/shotover-proxy/blob/main/shotover-proxy/examples/redis-cluster-ports-rewrite/docker-compose.yml) to run the Redis cluster.
```

Comment:
Doesnt sidecar refer to a specific functionality in containerized applications that we arent using here?

Reply:
Yeah I'm going in the process of Dockerizing Shotover to run a couple instances of it

Reply:
So is this not ready for review?

Reply:
Yes, I'll update it with the Docker changes once that gets merged.

Reply:
fair enough, since its not ready for review I marked as draft